src/badge.tsx

```diff
@@ -55,9 +64,11 @@ const Badge: React.FC<Props> = ({children, value, 'aria-label': ariaLabel}) => {
     }
 
     return (
-        <div className={classes.container} aria-label={ariaLabel}>
+        <div className={classes.container}>
             {value ? (
                 <div
+                    role="presentation"
+                    aria-hidden="true"
```

Comment:
isn't the aria-hidden redundant with role presentation? Why do we need both?

Reply:
voiceover was reading it even with role="presentation" :/